core/src/OpenACC/Kokkos_OpenACC_Team.hpp

```diff
@@ -134,17 +141,30 @@ class OpenACCTeamMember {
                     const int team_size,
                     const int vector_length)  // const TeamPolicyInternal<
                                               // OpenACC, Properties ...> & team
-      : m_team_size(team_size),
-        m_league_rank(league_rank),
+      : m_league_rank(league_rank),
         m_league_size(league_size),
+        m_team_size(team_size),
         m_vector_length(vector_length) {
 #ifdef KOKKOS_COMPILER_NVHPC
     m_team_rank = __pgi_vectoridx();
+#else
+#ifdef KOKKOS_COMPILER_CLANG
```

Comment:
```suggestion
#elif defined(KOKKOS_COMPILER_CLANG)
```
Presumably this never worked so please double check you need that change